admin/src/scenes/settings/CleSettings.tsx

```diff
@@ -0,0 +1,251 @@
+import SimpleToggle from "@/components/ui/forms/dateForm/SimpleToggle";
+import React from "react";
+import { Container } from "@snu/ds/admin";
+import { CohortDto } from "snu-lib/src/dto";
+import ToggleDate from "@/components/ui/forms/dateForm/ToggleDate";
+
+type CleSettingsData = Partial<CohortDto>;
```

Comment:
```suggestion
type CleSettings = Partial<CohortDto>;
```
pas sur que Data apporte quelques chose

Reply:
c'est déjà le nom du composant. Ceci dit, je serai pour ajouter le suffixe "component" au composant pour libérer le terme métier pour le modèle.

Reply:
pour ce composant il faudrait plutôt le suffixer avec sont utilité, ici Section: <CleSettingsSection />.

Pour les interface et type je préfixerai par `I` ça reste le plus claire

sinon ici en vrai ce ne sont pas des données liées aux CLE mais plutôt les settings en générale, pourquoi pas `IDynamiqueSettings` qui serait définit dans le parent ? (avec eventuellement in Pick ici)